spec/models/content_page_spec.rb

```diff
@@ -30,9 +30,13 @@
     expect { page.save! }.to_not raise_error
   end
 
-  it "sets the slug from the title, removing fullstops" do
```

Comment:
The full stop test still feels relevant and should be kept. What do you think?

Reply:
Done